src/korma/core.clj

```diff
@@ -487,11 +487,20 @@
      (= *exec-mode* :query) query
      (= *exec-mode* :dry-run) (do
                                 (println "dry run ::" sql "::" (vec params))
-                                (let [result-keys (conj (->> query :ent :rel vals
-                                                             (map deref)
-                                                             (filter (comp #{:belongs-to} :rel-type))
-                                                             (map :fk-key))
-                                                        (-> query :ent :pk))
+                                (let [fk-key (->> query :ent :rel vals
+                                                  (map deref)
+                                                  (filter (comp #{:belongs-to} :rel-type))
+                                                  (map :fk-key))
+                                      pk-key (-> query :ent :pk)
+                                      result-keys (concat
```

Comment:
updated :dry-run exec-mode to handle seq pk-key and fk-key, to be able to generate dry-runs for composite key related tests.